workshop/content/javascript/buildpipe/howto/_index.en.md

```diff
@@ -19,6 +19,6 @@ In this workshop, we are going to use the AWS Cloud Development Kit (also known
 
 That's right! You can describe your infrastructure by writing code in TypeScript, C#, Python or Java. Your code is then synthesized into CloudFormation and using the CDK CLI you can deploy it to an AWS account. 
 
-### How does SAM and CDK play together?
+### How do SAM and CDK play together?
```

Comment:
hmm interesting correction. For some reason `does` sounds better in my mind than `do` in this sentence. But English is not my first language, so I trust you better. :)